### ISSUE TITLE
Is it possible to have individual severity for rules in a list?

### ISSUE THREAD

**Issue by pmweeks98** [stylelint/stylelint#6406]
Is it possible to set individual severity for options within rules like *-disallowed-list?
For example below I would like to have a warning for the color rule but an error for the border rule

```yaml 
declaration-property-value-disallowed-list:
  /^border(?!.*style)/:
    - none
  /color/:
    - /var.*#[a-zA-Z0-9]{6}/
    - severity: warning
 ```

**Comment by jeddy3** [comment]
It's not. You can [write a plugin](https://stylelint.io/developer-guide/plugins) to cater to your specific needs, or use one written by the community, e.g. [`stylelint-declaration-strict-value`](https://github.com/AndyOGo/stylelint-declaration-strict-value) may fit your `/color/` needs.
